README.md

```diff
@@ -0,0 +1,106 @@
+  - *PersonalFinanceTool.pro*
+    - Qt project file which contains configuration flags, the included source and header files, and more.
+  - *README.md*
+    - This file which serves as supplemental material to help a user better understand how to obtain, install, and utilize the PFT.
+
+## Execution
+
+Assuming the prerequisites from above have been met and the source code has been cloned, the user should perform the following steps to run the PFT:
+
+1. Change directory into the *PersonalFinanceTool* folder.
+2. `qmake`
+3. `make clean`
+4. `make`
+5. `sudo make install`
+6. The user should see they now have a *bin* folder within the *PersonalFinanceTool* folder. Run the executable file inside that folder.
+
+If all was successful, the user should receive a pop-up welcoming them to the PFT.
+
+## Usage
+
+The following sections will detail various functionalities of the PFT.
+
+### Entering Total Amount Available
+
+Upon the initial run of the PFT, the user will receive a welcome pop-up, followed by a prompt to enter the total amount of money they have available in United States Dollars (USD).
+
+![Entering Total Amount Available](/images/total_amount_available_request.png)
+
+### Bill Entry
+
+Following the total amount available request, the user will be prompted to enter the bills they wish to keep track of via the bill widget. They'll be asked for the name of the bill, the amount due in USD, and the due date. When entering the due date, they can either manually type it in in *mm/dd/yyyy* format, or utilize the calendar view to select the date instead.
+
+![Bill Entry](/images/bill_entry.png)
+
+If the user does not enter a name for the bill and/or the amount due, they will receive an error message indicating there's missing bill information.
+
+![Missing Bill Information](/images/missing_bill_information.png)
+
+While entering bills, the user can click the *Close* button to terminate the application, the *Enter Another Bill* button to save the currently entered bill and relaunch the Bill Entry widget, and the *Done* button to save the currently entered bill and display the financial dashboard.
+
+### Configuration File
+
+As the user enters their bill information, the data entered will be written to a *.ini* configuration file. The file will be generated in a top level folder (*PersonalFinanceTool/config/*).
+
+```ini
+[Car]
+AmountDue=300.50
+DueDate=11/15/2024
+FundingStatus=Not Funded
+
+[FundsInformation]
+TotalFundsAvailable=100.00
+
+[Loans]
+AmountDue=200.00
+DueDate=11/28/2024
+FundingStatus=Not Funded
+
+[Rent]
+AmountDue=1200.00
+DueDate=11/1/2024
+FundingStatus=Not Funded
+```
+
+This file will be read on subsequent runs of the application to populate the bill table widget where the user can manage their bills (discussed in next section). The user can edit this information in the configuration file if they so choose, and the updated information will be used for the next run of the PFT.
```

Comment:
Add a little bit about how the configuration file allows the user to not have to re-enter their financial information on each run of the application due to the "saved state".

Reply:
Added.